### PR TITLE
Video listing page

### PR DESCRIPTION
**Pull request by aman11s**
Features added
- Theme
- Video listing page
- Filter by Category
- Responsiveness

### CODE REVIEW

File: src/components/ChipSection/ChipSection.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+      })}
+    </>
+  );
+};
+
+// {
+//   chipsList.map(({ id, name }) => {
+//     return <ChipSection key={id} name={name} />;
+//   });
+// }
```
Comment (Hardiegogo) [inline src/components/ChipSection/ChipSection.jsx]: Remove commented code, if not needed.

File: src/components/ChipSection/ChipSection.jsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+export const ChipSection = ({ selectCategory, setSelectCategory }) => {
+  return (
+    <>
+      {chipsList.map(({ id, name }) => {
+        return (
+          <div
+            onClick={() => setSelectCategory(name)}
+            className={`chip mx-2 ${selectCategory === name && "chip-active"}`}
+            key={id}
+          >
```
Comment (Hardiegogo) [inline src/components/ChipSection/ChipSection.jsx:20]: Good usage of object here.

**Review by Hardiegogo — APPROVED** [review]
Good structured code.
UI looks really cool.
Good to merge.✔

**Review by kart-c — APPROVED** [review]

**Review by SK02K1 — APPROVED** [review]
- Clean and modular code.
- Functionalities are working as expected.
- Clean UI

**Pull request merged**